modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -150,20 +151,72 @@
     /** */
     private File customDiagnosticDir;
 
+    /** */
+    private static final int serverNodeCnt = 2;
+    /** */
+    private static final int clientNodeCnt = 1;
+
+    /** */
+    protected static IgniteEx crd;
+    /** */
+    private static IgniteEx client;
+
+    /** {@inheritDoc} */
+    @Override protected void beforeTestsStarted() throws Exception {
+        crd = startGrids(serverNodeCnt);
+        client = startGrid("client");
+
+        crd.cluster().active(true);
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void afterTestsStopped() throws Exception {
+        GridTestUtils.cleanIdleVerifyLogFiles();
+
+        try (DirectoryStream<Path> files = newDirectoryStream(Paths.get(U.defaultWorkDirectory()), this::idleVerifyRes)) {
+            for (Path path : files)
+                delete(path);
+        }
+
+        stopAllGrids();
+
+        cleanPersistenceDir();
+    }
+
     /** {@inheritDoc} */
     @Override protected void beforeTest() throws Exception {
-        super.beforeTest();
+        sysOut = System.out;
 
-        initDiagnosticDir();
+        testOut = new ByteArrayOutputStream(16 * 1024);
+
+        checkpointFreq = DataStorageConfiguration.DFLT_CHECKPOINT_FREQ;
 
+        initDiagnosticDir();
         cleanDiagnosticDir();
     }
 
     /** {@inheritDoc} */
     @Override protected void afterTest() throws Exception {
-        super.afterTest();
+        log.info("Test output for " + currentTestMethod());
+        log.info("----------------------------------------");
 
-        cleanDiagnosticDir();
+        System.setOut(sysOut);
+
+        if (testOut != null)
+            System.out.println(testOut.toString());
+
+        testOut = null;
+
+        G.allGrids().stream().forEach(grid -> {
```

Comment:
destroy cache is a cluster wide operation. You could invoke destroy on coordinator only